### PR TITLE
Add salesperson to contacts

### PR DESCRIPTION
**Pull request by Stackpusher**
This Pull Request:

1. Adds a common salesperson object to represent salepersons for Quotes, Orders, Accounts, Contacts, and Facilities
2. Adds a SalespersonEncoder which will serialize an email address used to identify salespersons
3. Adds a SalespersonMapper
4. Adds salesperson to quotes and orders
5. Adds an example of how to use the sdk to assign a salesperson to a Contact

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -373,6 +374,16 @@ This will update the contact in Paperless Parts and refresh the local instance
     contact.create()
 ```
 
+### Adding a salesperson to a Contact
+```python
+    from paperless.objects.common import Salesperson
+    salesperson = Salesperson(email="sales@paperlessparts.com")
```
Comment (10flow) [inline README.md:380]: What happens if you use an email that doesn't correspond to a team member?

File: paperless/objects/quotes.py [hunk 2]
```diff
@@ -309,8 +302,9 @@ class Quote(
     revision_number: Optional[int] = attr.ib(
         validator=attr.validators.optional(attr.validators.instance_of(int))
     )
-    sales_person: SalesPerson = attr.ib(converter=convert_cls(SalesPerson))
-    estimator: SalesPerson = attr.ib(converter=convert_cls(SalesPerson))
+    sales_person: Salesperson = attr.ib(converter=convert_cls(Salesperson))
```
Comment (10flow) [inline paperless/objects/quotes.py:305]: For deprecating `sales_person`, at some point we should remove it from the SDK, while keeping it in the API response. That will force SDK users to remove the underscore when they upgrade SDK versions.

**Review by 10flow — COMMENTED** [review]
I would suggest adding a quick line to the README about what happens if you use an invalid Salesperson email address.

File: README.md (hunk 1 above)
Comment (Stackpusher) [reply README.md:380]: You will get a 400 with an error message "Invalid Salesperson".

**Review by 10flow — APPROVED** [review]

**Review by jeeppeck — APPROVED** [review]

**Pull request merged**